meta/runtime.yml

```diff
@@ -0,0 +1,2 @@
+---
+requires_ansible: '>=2.10.0'
```

Comment:
Ansible lint is complaining about this line.
```
meta-unsupported-ansible
requires_ansible key must be set to a supported version.
```

Reply:
i fixed it. Neede to be 2.11.0.